darts/utils/historical_forecasts/utils.py

```diff
@@ -661,3 +665,114 @@ def _get_historical_forecast_boundaries(
         hist_fct_fc_start,
         hist_fct_fc_end,
     )
+
+
+def _check_optimizable_historical_forecasts_global_models(
+    model,
+    forecast_horizon: int,
+    retrain: Union[bool, int, Callable[..., bool]],
+    show_warnings: bool,
+    allow_autoregression: bool,
+) -> bool:
+    """
+    Historical forecast can be optimized only if `retrain=False` and `forecast_horizon <= model.output_chunk_length`
+    (no auto-regression required).
+    """
+
+    supported_retrain = (retrain is False) or (retrain == 0)
```

Comment:
The name of this variable is a bit unclear, I would maybe call it something like `retrain_off` or `retrain_expected`.